### PR TITLE
Add instructions for installing go-bindata

### PR DESCRIPTION
**Pull request by SataQiu**
**What type of PR is this?**
/kind cleanup
/kind documentation

**What this PR does / why we need it**:
Add instructions for installing go-bindata.

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:
```release-note
NONE
```

### CODE REVIEW

File: hack/generate-bindata.sh [hunk 1]
```diff
@@ -33,7 +33,8 @@ fi
 export PATH="${KUBE_OUTPUT_BINPATH}:${PATH}"
 
 if ! which go-bindata &>/dev/null ; then
-	echo "Cannot find go-bindata."
+	echo 'Can not find go-bindata, install with:'
```
Comment (ixdy) [inline hack/generate-bindata.sh]: hm, we should be using vendored `go-bindata`. `make generated_files` will build/install `go-bindata`, but I guess this script is maybe called from other places?

**Comment by spiffxp** [comment]
/priority important-longterm
/sig testing
because go-bindata

/hold
sounds like the instructions here are incorrect?

File: hack/generate-bindata.sh (hunk 1 above)
Comment (SataQiu) [reply hack/generate-bindata.sh]: I followed the steps described by `translations/README.md`, and "Cannot find go-bindata" error occurred when executing `./hack/update-translations.sh`. 
The script `update-translations.sh` ends up calling `generate-bindata.sh`. 
I tried to execute `make generated_files` firstly, but this did not install `go-bindata` to `$GOPATH/bin`. 
Should we guide users to install go-bindata more specifically?
Like:
```
echo 'Can not find go-bindata, install with:'
echo 'go build -o $GOPATH/bin/go-bindata ./vendor/github.com/jteeuwen/go-bindata/go-bindata'
```

**Pull request closed without merging**